examples/localai-webui/README.md

```diff
@@ -0,0 +1,26 @@
+# localai-webui
+
+Example of integration with [dhruvgera/localai-frontend](https://github.com/Dhruvgera/LocalAI-frontend).
+
+![image](https://user-images.githubusercontent.com/42107491/235344183-44b5967d-ba22-4331-804c-8da7004a5d35.png)
+
+## Setup
+
+```bash
+# Clone LocalAI
+git clone https://github.com/go-skynet/LocalAI
+
+cd LocalAI/examples/localai-webui
+
+# (optional) Checkout a specific LocalAI tag
+# git checkout -b build <TAG>
+
+# Download gpt4all-j to models/ in the parent LocalAI project dir
+wget https://gpt4all.io/models/ggml-gpt4all-j.bin -O ../../models/ggml-gpt4all-j
```

Comment:
the commands here are not working, there are no models inthe instance. Did you forgot maybe to check out with git the symlink to the `models` folder?

Reply:
This is just a sample command to show where the models should be downloaded, I'm not shipping any in the container